### ISSUE TITLE
Settings per project / sln

### ISSUE THREAD

**Issue by JohanLarsson** [Xavalon/XamlStyler#8]
When working in a team shared formatting rules that can be put in version control with the code would be nice.

I'm thinking something like
```
Foo.sln
Foo.sln.xamlstyle
  Bar.csproj
  Bar.csproj.xamlstyle
```

**Comment by grochocki** [comment]
There are a number of features I am backporting from [XAML Magic](https://github.com/grochocki/XamlMagic), including network-based configurations for teams. Local configurations also seem like a logical extension of this.

**Comment by grochocki** [comment]
Implemented by #46
